### PR TITLE
BD-2591 | Migrating all legacy sections into new 'Legacy SDKs' directory

### PR DESCRIPTION
**Pull request by internetisaiah**
# Pull Request/Issue Resolution

#### Description of Change:

Migrating all legacy sections into new 'Legacy SDKs' directory and adding `noindex: true` to all Markdown files with the new 'Legacy SDKs' directory.

- [Jira Ticket](https://jira.braze.com/browse/BD-2591)

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @bre-fitzgerald as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] If the documentation involves a 1) paid SKU, 2) a third party, 3) SMS, 4) AI, or 5) privacy, ensure that Maria Maldonado on the Legal team has signed off.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @bre-fitzgerald for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs<br>Currents</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Intelligence<br>In-App Messages<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging Experience<br>Message Components<br>Email (Composition and Infrastructure) (tag @rachel-feinberg for Liquid use cases)</td>
  </tr>
  <tr>
    <td>@rachel-feinberg</td>
    <td>Customer Lifecycle, Identity and Permissions<br>SMS<br>User Targeting<br>Reporting</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/legacy_sdks/ios/advanced_use_cases/google_tag_manager.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ platform: iOS
 page_order: 7
 description: "This article covers how to initialize, configure, and implement the Google Tag manager into your iOS app."
 
+noindex: true
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/legacy_sdks/ios/advanced_use_cases/google_tag_manager.md:8]: Curious, but what does `noindex: true` do?

**Review by josh-mccrowell-braze — APPROVED** [review]
Looks good to me!

**Comment by josh-mccrowell-braze** [comment]
I suspect that making the legacy pages `noindex` will be an effective (if blunt force!) fix for them appearing in search, but if the Swift articles are still not being found by the search, you can experiment with changing their `search_rank` per the [Confluence documentation](https://confluence.braze.com/pages/viewpage.action?pageId=248222285). I'd advocate for this being a discrete, separate task, though.

File: _docs/_developer_guide/platform_integration_guides/legacy_sdks/ios/advanced_use_cases/google_tag_manager.md (hunk 1 above)
Comment (internetisaiah) [reply _docs/_developer_guide/platform_integration_guides/legacy_sdks/ios/advanced_use_cases/google_tag_manager.md:8]: Hey @hokstuff ,

Great questions! The `noindex: true` sets the page as hidden in the Braze Docs search bar (like in your image above). As I currently understand it, we aren't able to preview search bar changes in our deployment environment (even if the tags are properly added). So, once this PR is merged these docs _should_ be hidden form the search results--as that is my intention.

However, if they are still showing up even after the PR is merged, I'll be sure to prioritize a fix asap as that is apart of my original goal for this PR.

Please let me know if you have any other questions!

**Comment by internetisaiah** [comment]
> I suspect that making the legacy pages `noindex` will be an effective (if blunt force!) fix for them appearing in search, but if the Swift articles are still not being found by the search, you can experiment with changing their `search_rank` per the [Confluence documentation](https://confluence.braze.com/pages/viewpage.action?pageId=248222285). I'd advocate for this being a discrete, separate task, though.

Thanks Josh! I'll merge this PR and see if the fix worked. If not, I'll use your resources for a new fix.

**Pull request merged**